src/ingest_validation_tools/enums.py

```diff
@@ -42,17 +42,16 @@
         'MxIF',
         'MALDI-IMS',
         # 'MS (shotgun lipidomics)',
+        'Multiplex Ion Beam Imaging',
         # 'NanoDESI',
         # 'NanoPOTS',
         'PAS microscopy',
-        # 'scATACseq',
         'sciATACseq',
         'sciRNAseq',
         'seqFISH',
         # 'SIMS-IMS',
         'SNARE-seq2',
         'snATACseq',
-        # 'snRNA',
```

Comment:
This is odd: the string `snRNAseq` is used in the schemas, but not just `snRNA`. (We used to check for consistency between these lists, but that was disabled to make Joel's work easier.)

Same kind of question: What is the right course, and what do we do to make schemas consistent with the global list of allowed names?

Reply:
@mccalluc I believe the cross-checking mechanism that was in place before you disabled it verified that all names in the schemata existed in the enum, but not the other way around.  snRNA existed in the enum only.

Reply:
👍